PackageViewModel/PackageViewModel.cs

```diff
@@ -110,6 +110,9 @@ internal ISettingsManager SettingsManager
             get { return _settingsManager; }
         }
 
+        public event EventHandler BeforeSave;
```

Comment:
Required to ignore file watcher changes during save... Or is there an other way to do it?

Reply:
What is the purpose of using a file watcher here?

Reply:
To detect package file renames / deletes / changes so we can either dispose or reload the view model.